doc/content/the-things-stack/host/docker/certificates.md

```diff
@@ -139,3 +108,50 @@ config/
 ```
 
 Be sure to configure `docker-compose.yml` and `ttn-lw-stack-docker.yml` for your custom certificates, as shown in [using custom certificates](#using-custom-certificates).
+
+## Using Custom Certificates
+
+To use CA certificates you already have or [self-signed certificates](#custom-certificate-authority), you will need to uncomment the custom certificates section of `docker-compose.yml`:
+
+{{< highlight yaml "linenos=table,linenostart=66" >}}
+{{< readfile path="/content/the-things-stack/host/docker/configuration/docker-compose-custom-certificates.yml" from=66 to=79 >}}
+{{< /highlight >}}
+
+You will also need to comment out the Let's Encrypt section of `ttn-lw-stack-docker.yml`:
+
+{{< highlight yaml "linenos=table,linenostart=48" >}}
+{{< readfile path="/content/the-things-stack/host/docker/configuration/ttn-lw-stack-docker-custom-certificates.yml" from=48 to=55 >}}
+{{< /highlight >}}
+
+And uncomment the custom certificates section:
+
+{{< highlight yaml "linenos=table,linenostart=41" >}}
+{{< readfile path="/content/the-things-stack/host/docker/configuration/ttn-lw-stack-docker-custom-certificates.yml" from=41 to=46 >}}
+{{< /highlight >}}
+
+Finally you must set the correct permissions, as shown in [Certificates from a Certificate Authority](#certificates-from-a-certificate-authority)
+
+### Certificates from a Certificate Authority
```

Comment:
Why is this under a separate title? Same steps apply to both custom CA certificates and self signed certificates afaik, so it can be under **Using custom certificates**

Reply:
You are correct, they can just be combined. The "Certificates from a Certificate Authority" was a seperate section before too, so i wasn't sure if i should just combine it.